src/components/experiment/__tests__/Experiment.text.tsx

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Experiment from '../Experiment';
```

Comment:
You should have some unit tests in here.

Reply:
Added, as well as tests for ExperimentItem